test/test.jl

```diff
@@ -395,3 +395,9 @@ end
 end
 @test_throws ErrorException @testset "$(error())" begin
 end
+
+io = IOBuffer()
+@test (print(io, Base.Test.Error(:test_error, "woot", 5, backtrace())); 1) == 1
```

Comment:
The stdio streams are redirected above 79 and trying to do something like `readstring(rdo)` never returns. I am kinda a noob with streams.